contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
 </ComparisonTable>
 
-#### Import
-  
-<ComparisonTable column1="Amplitude Analytics" column2="PostHog">
-  <ComparisonRow column1={true} column2={true} feature="Redshift" />
-  <ComparisonRow column1={true} column2={true} feature="Google Cloud Storage" />
-  <ComparisonRow column1={true} column2={true} feature="Snowflake" />
-  <ComparisonRow column1={true} column2={true} feature="Amazon S3" />
-  <ComparisonRow column1={true} column2={true} feature="Segment" />
-  <ComparisonRow column1={false} column2={true} feature="Sentry" />
-  <ComparisonRow column1={true} column2={true} feature="Zendesk" />
-  <ComparisonRow column1={true} column2={true} feature="API" />
+### Integrations
+
+PostHog and Amplitude both offer a wide range of integrations, sources and destinations. PostHog offers more integrations with developer tools, such as Sentry, Zapier and Zendesk, while Amplitude has more integrations with marketing and advertising platforms. See our [data pipeline docs](/docs/cdp) for a full list of PostHog integrations.
+
+<ComparisonTable column1="PostHog" column2="Amplitude">
+   <ComparisonRow column1={true} column2={true} feature="Imports" description="Import data from data warehouses and other sources" />
+   <ComparisonRow column1={true} column2={true} feature="Exports" description="Export data to data warehouses other destinations" />
+   <ComparisonRow column1={true} column2={true} feature="Segment" description="Send events via Segment" />   
+   <ComparisonRow column1={true} column2={false} feature="Zapier" description="Trigger Zapier automations" />
+   <ComparisonRow column1={true} column2={false} feature="Sentry" description="Connect to Sentry data" />
+   <ComparisonRow column1={true} column2={false} feature="Zendesk" description="Two-way integration for customer support" />
+   <ComparisonRow column1={true} column2={true} feature="Slack" description="Alerts for Slack" />
+   <ComparisonRow column1={true} column2={true} feature="Microsoft Teams" description="Alerts for Microsoft Teams" />
+   <ComparisonRow column1={true} column2={false} feature="Community integrations" description="Build your own integration" />
+   <ComparisonRow column1={false} column2={true} feature="Google Ads" description="Import ROI data from Google Ads" />   
 </ComparisonTable>
 
-## Strengths of PostHog
+### Pricing
 
-![PostHog screenshot](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
+**PostHog** charges for actual usage – i.e. product analytics events, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier** limits:
 
-### Integrated session replay
+|                       | **Free usage per month** 
+|-----------------------|--------------------------
+| **Product analytics** | 1 million events        
+| **Session replay**    | 5,000 recordings         
```

Comment:
might have to update mentions of this on soon.

Reply:
This is the updated figure, afaik. Used to be 15k